actions_helper/commands/create_task_definition.py

```diff
@@ -0,0 +1,98 @@
+from typing import Any
+
+import click
+from botocore.client import BaseClient
+
+from actions_helper.commands.get_active_task_definition_by_tag import get_active_task_definition_arn_by_tag
+from actions_helper.outputs import CreateTaskDefinitionOutput
+from actions_helper.utils import PLACEHOLDER_TEXT, set_error
+
+KEYS_TO_DELETE_FROM_TASK_DEFINITION = (
+    "taskDefinitionArn",
+    "revision",
+    "status",
+    "requiresAttributes",
+    "compatibilities",
+    "registeredAt",
+    "registeredBy",
+)
+
+
+def get_rendered_task_definition(ecs_client: BaseClient, task_definition_arn: str, image_uri: str) -> dict[str, Any]:
+    task_definition = ecs_client.describe_task_definition(taskDefinition=task_definition_arn)["taskDefinition"]
+
+    try:
+        (container_image,) = {
+            container_definition["image"] for container_definition in task_definition["containerDefinitions"]
+        }
+        if container_image != PLACEHOLDER_TEXT:
+            set_error(f"Not all values for containerDefinitions 'image' equal to '{PLACEHOLDER_TEXT}'")
+    except ValueError:
+        set_error("Only a single image name is allowed")
+
+    for container_definition in task_definition["containerDefinitions"]:
+        container_definition["image"] = image_uri
+
+    for key in KEYS_TO_DELETE_FROM_TASK_DEFINITION:
+        del task_definition[key]
+
+    return task_definition
+
+
+def deploy_and_tag_task_definition(
+    ecs_client: BaseClient,
+    task_definition: dict[Any, Any],
```

Comment:
```suggestion
    task_definition: dict[str, Any],
```
matching the types from `get_redenderd_task_definition`?